pkg/cmd/cli/cmd/deploy.go

```diff
@@ -228,6 +229,7 @@ func (o DeployOptions) deploy(config *deployapi.DeploymentConfig, out io.Writer)
 	_, err = o.osClient.DeploymentConfigs(config.Namespace).Update(config)
 	if err == nil {
 		fmt.Fprintf(out, "Started deployment #%d\n", config.Status.LatestVersion)
+		fmt.Fprintf(out, "Use '%s logs -f dc/%s' to track its progress.\n", o.baseCommandName, config.Name)
```

Comment:
This is also in line with what we suggest in new-app:
https://github.com/openshift/origin/blob/185cefad6da5ef7ef1eb1c581bd186d4b2e491f7/pkg/cmd/cli/cmd/newapp.go#L314. I can have it in one line as is, not sure if it can be shorter though